show/reboot_cause.py

```diff
@@ -8,7 +8,7 @@
 import utilities_common.cli as clicommon
 
 
-PREVIOUS_REBOOT_CAUSE_FILE_PATH = "/host/reboot-cause/previous-reboot-cause.json"
+PREVIOUS_REBOOT_CAUSE_FILE_PATH = "/host/reboot-cause/reboot-cause.txt"
```

Comment:
Why change this? Who will generated the `previous-reboot-cause.json` file?

I saw the UT were introduced here: https://github.com/Azure/sonic-utilities/pull/1210/files
@sujinmkang Could you please join the review?

Reply:
I just found a piece of code in sonic-buildimage regarding reboot-cause that I left out. Though reboot-cause still gives "Unknown", I think my fix is only a workaround to passing the test. I will close this pr and create a better solution.